cypress/integration/adminFlows/pages/landingPage.spec.js

```diff
@@ -4,23 +4,28 @@ describe('Set a landing page from the admin portal', () => {
     cy.fixture('users/adminUser.json').as('user');
     cy.get('@user').then((user) => {
       cy.loginAndVisit(user, '/admin/customization/config').then(() => {
-        // Ensure Forem instance is private
-        // NOTE: @citizen428 - We may need to find a better situation for this
-        // long-term.
+        cy.get('#new_settings_user_experience').as('userExperienceSectionForm');
+        //   // Ensure Forem instance is private
+        //   // NOTE: @citizen428 - We may need to find a better situation for this
+        //   // long-term.
```

Comment:
Refactor (non-blocking): Looks like the comments are commented out. 😎

```suggestion
        cy.get('#new_settings_user_experience').as('userExperienceSectionForm');
        // Ensure Forem instance is private
        // NOTE: @citizen428 - We may need to find a better situation for this
        // long-term.
```